box.py

```diff
@@ -213,10 +213,10 @@ def __init__(self, *args, **kwargs):
             'modify_tuples_box': kwargs.pop('modify_tuples_box', False)
             }
         if len(args) == 1:
-            if isinstance(args[0], basestring):
+            if isinstance(args[0], str):
                 raise ValueError('Cannot extrapolate Box from string')
             if isinstance(args[0], Mapping):
-                for k, v in args[0].items():
+                for k, v in list(args[0].items()):
```

Comment:
2to3 casts all changes from `list` to `iter` items as lists just in case. However in all these cases, it is appropriate to use `.items` an iterable.